tests/testthat/test_a_is_workdir_clean.R

```diff
@@ -2,26 +2,28 @@ test_that("is_workdir_clean", {
   path <- tempfile("clean")
   dir.create(path)
   on.exit(unlink(path, recursive = TRUE), add = TRUE)
-  repo <- init(path)
-  git2r::config(repo = repo, user.name = "junk", user.email = "junk@inbo.be")
+
+  gert::git_init(path = path)
```

Comment:
Here you only need the `package::function` syntax when you need a function that `checklist` does not import. Hence `git2r::config()` but not `git2r::add()`.

Reply:
Done in 1563e1e